src/Tasks/Microsoft.NET.Build.Tasks/GenerateToolsSettingsFile.cs

```diff
@@ -4,11 +4,14 @@
 using Microsoft.Build.Framework;
 using System.Xml.Linq;
 using System.IO;
+using System;
 
 namespace Microsoft.NET.Build.Tasks
 {
     public class GenerateToolsSettingsFile : TaskBase
     {
+        private static readonly Version _formatVersion = new Version(1,0,0);
```

Comment:
Can you comment on when this is expected to change?

Reply:
will do